proposals/views/proposal_views.py

```diff
@@ -649,19 +656,15 @@ class ProposalUpdatePreAssessment(PreAssessmentMixin, ProposalUpdate):
 
 
 class ProposalSubmitPreAssessment(ProposalSubmit):
-    def form_valid(self, form):
+    def submit_proposal(self, form):
         """
         Performs actions after saving the form
         - Save the preassessment PDF on the Proposal
-        - End the preassessment phase
+        - Start the preassessment review
         """
-        # Note that the below method does NOT call the ProposalSubmit method, as that would generate the full PDF.
-        success_url = super(ProposalSubmitPreAssessment, self).form_valid(form)
-        if 'save_back' not in self.request.POST and 'js-redirect-submit' not in self.request.POST:
-            proposal = self.get_object()
-            proposal.generate_pdf()
-            start_review_pre_assessment(proposal)
-        return success_url
+        proposal = self.get_object()
+        proposal.generate_pdf()
+        start_review_pre_assessment(proposal)
```

Comment:
While I do like this approach, I do wonder if it might not be better to move all 'review-start' logic into `start_review`.

That util function is already a 'switch-function' that calls other functions depending on what the actual situation is. Adding the extra conditions to that function might make more sense, then we can just use the same code for every submit everywhere else. 
Especially as currently the only difference between the two variants is calling a different review-start functions/conditions.